EMML/1001-2000/EMML1832.xml

```diff
@@ -45,35 +146,930 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+                           children, who are with them, before <persName ref="PRS5299Herod"/>, who is seated on his throne 
+                           and <ref type="authFile" corresp="AT1020Massacre"/>.</desc>
+                        <q xml:lang="gez">ኀበ፡ ቀተለ፡ ሄሮድስ፡ ሕፃናተ፡          ራሔል፡ ትኬልሕ፡ በእንተ፡ ደቂቃ፡ </q>
+                     </decoNote>
+                     <decoNote xml:id="d20" type="miniature">
+                        <locus target="#20r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1011Entry"/>.</desc>
+                        <q xml:lang="gez">ዘኬዎስ፡ በኣተ፡ ኢየሩሳሌም። </q>
+                     </decoNote>
+                     <decoNote xml:id="d21" type="miniature">
+                        <locus from="20v" to="21r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1032Washing"/>.</desc>
+                        <q xml:lang="gez">ኀበ፡ ኀፀበ፡ እግረ፡ አርዳኢሁ፡</q>
+                     </decoNote>
+                     <decoNote xml:id="d22" type="miniature">
+                        <locus target="#21v"/>
+                        <desc>Miniature depicting <persName ref="PRS6819Mary"/> weeping, <persName ref="PRS5695John"/> looking from 
+                           afar and <ref type="authFile" corresp="AT1006Denial"/>.</desc>
+                        <q xml:lang="gez">ዶርሆ፡     ዶርሆ፡     ማርያም፡ ላሐወት፡       ዮሐንስ፡ እምርሑቅ፡ ይቀውም፡      ጴጥሮስ፡ ክሕደ። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d23" type="miniature">
+                        <locus target="#22r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1003Arrest"/>.</desc>
+                        <q xml:lang="gez">ኀበ፡ አኀዝዎ፡ አይሁድ፡ ለኢየሱስ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d24" type="miniature">
+                        <locus target="#22v"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1005Crucifixion">Crucifixion</ref> showing the two thieves, 
+                           the two soldiers and a ram at the top of the cross, but not Christ on the cross.</desc>
+                        <q xml:lang="gez">ፉያታይ፡       ፈያታዊ፡       <hi rend="rubric">ዘርአ፡ ዘኃበሁ፡</hi>    <hi rend="rubric">ዘአስተዮባሐለት፡</hi></q>
+                     </decoNote>
+                     <decoNote xml:id="d25" type="miniature">
+                        <locus target="#23r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1033Ascension"/> and <persName ref="PRS6819Mary"/> 
+                           with <persName ref="PRS11430Apostles"/>.</desc>
+                        <q xml:lang="gez">ኀበ፡ ዓርገ፡ ክርስቶስ። ።       ሐዋርያት፡ ምስለ፡ ማርያም፡ ቅድስት። </q>
+                     </decoNote>
+                     <decoNote xml:id="d26" type="miniature">
+                        <locus target="#35v"/>
+                        <desc>Miniature depicting <persName ref="PRS6902Matthew"/>.</desc>
```

Comment:
as @gnizla remarked here https://github.com/BetaMasaheft/Documentation/issues/2453#issuecomment-1900401931 you could add `<ref type="authFile" corresp="AT1013EvaPortrait"/>`for Evangelist's portraits here and below; also `AT1046Apostle ` `AT1035HolyMan`